Presentation.Web/Controllers/API/V2/External/ItInterfaces/ItInterfaceV2Controller.cs

```diff
@@ -234,6 +234,7 @@ public IHttpActionResult DeactivateItInterfaceAsRightsHolder([NonEmptyGuid] Guid
             bool? includeDeactivated = null,
             DateTime? changedSinceGtEq = null,
             string nameEquals = null,
+            [NonEmptyGuid] Guid? usedInOrganizationUuid = null,
```

Comment:
no documentation for the parameter.... 
![image](https://user-images.githubusercontent.com/13116889/222425019-2ded661d-1347-4555-9304-46e74ee1b132.png)